llvm/lib/SYCLLowerIR/SYCLPropagateAspectsUsage.cpp

```diff
@@ -410,6 +410,7 @@ void propagateAspectsThroughCG(Function *F, CallGraphTy &CG,
 ///  - checks if function has "!sycl_used_aspects" metadata
```

Comment:
Perhaps we should update this comment to say that this function also collects aspects declared through an attribute/property